src/DGtal/dec/DiscreteExteriorCalculus.h

```diff
@@ -400,6 +401,17 @@ namespace DGtal
     LinearOperator<Self, 0, duality, 0, duality>
     laplace() const;
 
+    /**
+     * Convolutional Laplace operator from duality 0-forms to duality 0-forms. See \ref secHowToHeatLaplace.
+     * @param h the grid step
```

Comment:
please add a cite and an entry in the bibtex pointing to your DGCI paper.

Reply:
from dual 0-forms to dual 0-froms

Reply:
the operator is valid from any duality